test/tests/async_get_handler_test.js

```diff
@@ -0,0 +1,54 @@
+import Router from 'router';
+import { Promise } from "rsvp";
+
+QUnit.module('Async Get Handler', {
```

Comment:
Can you add a comment here to explain that we are intentionally using `QUnit.module` to avoid the async shenanigans in the `module` that is imported in the other tests?